fail2ban/setup.py

```diff
@@ -32,7 +32,7 @@ def updatePyExec(bindir, executable=None):
 		executable = sys.executable
 	pypath = os.path.join(bindir, 'fail2ban-python')
 	# if not exists or point to another version - update link:
-	isfile = os.path.isfile(pypath)
+	isfile = os.path.isfile(os.path.realpath(pypath))

```

Comment:
"isfile" is stricter as "exists", should it point (however) to a directory...
And dereferencing makes sure that the target exists (because link may point to nonexistent target)